todoApp/spec/main.tsp

```diff
@@ -118,6 +113,19 @@ model TodoUrlAttachment {
   url: url;
 }
 
+@jsonSchema
+model TodoAttachment {
+  /** The file name of the attachment */
+  @maxLength(255)
+  filename: string;
+
+  /** The media type of the attachment */
+  mediaType: string;
+
+  /** The contents of the file */
+  contents: bytes;
+}
+
 @jsonSchema
 model TodoFileAttachment is File;
```

Comment:
Mark seems saying in meeting that we no longer need `TodoFileAttachment`, and can directly put `HttpPart<File>` in `ToDoItemMultipartRequest` and `FileAttachmentMultipartRequest`. Did anyone remember same, or it is just me?